src/main/java/org/sonarlint/intellij/SonarLintIcons.kt

```diff
@@ -82,12 +82,42 @@ object SonarLintIcons {
     @JvmField
     val FOCUS = getIcon("/images/focus.svg")
 
+    private val BUG_ICONS = mapOf(
+        IssueSeverity.BLOCKER to getIcon("/images/bug/bugBlocker.svg"),
+        IssueSeverity.CRITICAL to getIcon("/images/bug/bugHigh.svg"),
+        IssueSeverity.MAJOR to getIcon("/images/bug/bugMedium.svg"),
+        IssueSeverity.MINOR to getIcon("/images/bug/bugLow.svg"),
+        IssueSeverity.INFO to getIcon("/images/bug/bugInfo.svg")
+    )
+
+    private val CODE_SMELL_ICONS = mapOf(
+        IssueSeverity.BLOCKER to getIcon("/images/codeSmell/codeSmellBlocker.svg"),
+        IssueSeverity.CRITICAL to getIcon("/images/codeSmell/codeSmellHigh.svg"),
+        IssueSeverity.MAJOR to getIcon("/images/codeSmell/codeSmellMedium.svg"),
+        IssueSeverity.MINOR to getIcon("/images/codeSmell/codeSmellLow.svg"),
+        IssueSeverity.INFO to getIcon("/images/codeSmell/codeSmellInfo.svg")
+    )
+
+    private val VULNERABILITY_ICONS = mapOf(
+        IssueSeverity.BLOCKER to getIcon("/images/vulnerability/vulnerabilityBlocker.svg"),
+        IssueSeverity.CRITICAL to getIcon("/images/vulnerability/vulnerabilityHigh.svg"),
+        IssueSeverity.MAJOR to getIcon("/images/vulnerability/vulnerabilityMedium.svg"),
+        IssueSeverity.MINOR to getIcon("/images/vulnerability/vulnerabilityLow.svg"),
+        IssueSeverity.INFO to getIcon("/images/vulnerability/vulnerabilityInfo.svg")
+    )
+
+    private val PROBABILITY_ICONS = mapOf(
+        VulnerabilityProbability.HIGH to getIcon("/images/hotspot/hotspotHigh.svg"),
+        VulnerabilityProbability.MEDIUM to getIcon("/images/hotspot/hotspotMedium.svg"),
+        VulnerabilityProbability.LOW to getIcon("/images/hotspot/hotspotLow.svg")
+    )
+
     private val SEVERITY_ICONS = mapOf(
-        IssueSeverity.BLOCKER to getIcon("/images/severity/blocker.svg"),
-        IssueSeverity.CRITICAL to getIcon("/images/severity/critical.svg"),
-        IssueSeverity.INFO to getIcon("/images/severity/info.svg"),
-        IssueSeverity.MAJOR to getIcon("/images/severity/major.svg"),
-        IssueSeverity.MINOR to getIcon("/images/severity/minor.svg")
+        IssueSeverity.BLOCKER to getIcon("/images/impact/blocker.svg"),
+        IssueSeverity.CRITICAL to getIcon("/images/impact/high.svg"),
+        IssueSeverity.MAJOR to getIcon("/images/impact/medium.svg"),
```

Comment:
We could align the image name i.e major ->impact/major

Reply:
Actually, the IssueSeverity shares the same icon as the ImpactSeverity (Standard/MQR mode), so I cannot really have the same names for both 😄

Reply:
We could duplicate the images and store them under /images/severity/ folder so that if we need to change them in the future we would be confident it won't break the other icon